web3/rpc/chainlist/chainlist.go

```diff
@@ -16,6 +16,7 @@ const (
 	jsonRPCVersion = "2.0"
 
 	requiredEthLogsBlocks = 9990 // Number of blocks to check for eth_getLogs
+	sepoliaShortName      = "sep"
```

Comment:
Instead of doing a specific case only for sepolia, better create a `map["networkName"] => "chainListName"`
So you can do `newName, ok := networkNameReplaceMap["sepolia"]` and use it only if exists. Ensure that all network names that we have defined are either correct on chainList and if not, create the corresponding entries in the map. 

Place the map into `chainlist/networknames.go`